main/src/main/java/cgeo/geocaching/connector/gc/GCConstants.java

```diff
@@ -95,11 +95,11 @@ public final class GCConstants {
 
     //Try to find pattern as follows:
     //"publicGuid":"abc.def","referenceCode":"PRxyz","id":1234567,"username":"user","dateCreated":"2012-01-21T20:51:14","findCount":15123,
-    public static final Pattern PATTERN_LOGIN_NAME_CACHE_COUNT = Pattern.compile("\"publicGuid\":\"[^\"]+\",\"referenceCode\":\"[^\"]+\",\"id\":[0-9]+,\"username\":\"([^\"]+)\",\"dateCreated\":\"[0-9:T-]+\",\"findCount\":([0-9]+),");
+    public static final Pattern PATTERN_LOGIN_NAME1 = Pattern.compile(",\"referenceCode\":\"[^\"]+\",\"id\":[0-9]+,\"username\":\\s*\"(([^\\\\\\\"]*(\\\\[a-z\\\"])*)*)\\\",");
     public static final Pattern PATTERN_FINDCOUNT = Pattern.compile("\"findCount\":\\s*([0-9]+)[,\\s]");
```

Comment:
I have an idea to recognise that we are in the right place (block), but without the need for an ordered position (reference to another entry).
The block starts with `window['chromeSettings'] = {`, so we could use something like
`window\['chromeSettings'\].*"username":\s*\"(([^\\\\\"]*(\\[a-z\"])*)*)\"`
Maybe additional escapes are necessary.